### PR TITLE
updated the terms.json file

### PR DESCRIPTION
**Pull request by 0xTife**

### CODE REVIEW

**Review by Qudusayo — CHANGES_REQUESTED** [review]
- FUD and REKT already exist in the terms, You should update them instead
https://github.com/Olanetsoft/web3terms/blob/48ef57a0c805f1c15777e065614ca5945813bc6c/data/terms.json#L355-L356
https://github.com/Olanetsoft/web3terms/blob/48ef57a0c805f1c15777e065614ca5945813bc6c/data/terms.json#L603-L604

- You missed out on sorting the terms [as mentioned here](https://github.com/Olanetsoft/web3terms#-after-making-a-change) before making a PR.

File: data/terms.json [hunk 1]
```diff
@@ -758,5 +758,9 @@
   {
     "term": "Yield Farming",
     "meaning": "Yield farming is the process of using decentralized finance (DeFi) to maximize returns. Users lend or borrow crypto on a DeFi platform and earn cryptocurrency in return for their services."
-  }
+  },
+  {
+    "term": "Hash Rate",
```
Comment (Olanetsoft) [inline data/terms.json:763]: This should be in lower case

File: data/terms.json [hunk 2]
```diff
@@ -758,5 +758,9 @@
...
+    "meaning": "Measurement of performance that reveals how many hashes per second your computer is capable of producing. Each hash is an attempt to find a block by creating a unique block candidate and testing it against the network."
```
Comment (Olanetsoft) [inline data/terms.json:764]: Ensure this does not exceed 279 characters

**Review by Olanetsoft — APPROVED** [review]

**Comment by Olanetsoft** [comment]
@gbolame Please respond to feedback; let's close this! Thank you

**Pull request closed without merging**